Kajabi/sage-lib
# Sage: Integrate Pine into icons

## Description
Updates components and documentation to use `pds-icon`. Switching over the new icon system should help to eliminate the current manual process.

## Screenshots
Minimal visual changes should be expected.


## Testing in `sage-lib`

- Navigate to each component for [Rails](http://localhost:4000/pages/index) & [React](http://localhost:4100/?path=/docs/welcome-to-sage--page)
- Verify icons render as expected.
- Verify icons render in docs as expected


## Testing in `kajabi-products`

1. (**BREAKING**) Converts icon font to pds-icons in components and documentation.
   - [ ] A full regression of the kajabi admin should be performed to make sure all icons still display as expected.


## Related
[DSS-623](https://kajabi.atlassian.net/browse/DSS-623)


[DSS-623]: https://kajabi.atlassian.net/browse/DSS-623?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
